data/styles/clear/include/Icons.mapcss

```diff
@@ -1948,9 +1952,15 @@ node|z17-[emergency=phone]
 {icon-image: phone-m.svg;icon-min-distance: 30;}
 
 node|z17-[amenity=recycling],
-area|z17-[amenity=recycling]
+area|z17-[amenity=recycling],
+node[amenity=waste_transfer_station],
+area[amenity=waste_transfer_station],
 {icon-image: recycling-m.svg;icon-min-distance: 30;}
 
+node|z17-[amenity=sanitary_dump_station],
+area|z17-[amenity=sanitary_dump_station],
```

Comment:
Let's make it visible at lower zoom levels.
E.g. `amenity=fuel` is visible from z14 - dump stations are also used by motorists and its not a lot of them and usually they're located off dense areas like city centers so they won't clog up the map -  I suggest making it z15- at least.